test/utils/Modifiers.t.sol

```diff
@@ -0,0 +1,529 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity >=0.8.22;
+
+import { ISablierLockup } from "src/core/interfaces/ISablierLockup.sol";
+
+import { Utils } from "./Utils.sol";
+
+abstract contract Modifiers is Utils {
```

Comment:
correct it works for me
the only thing is that i remember @PaulRBerg making an argument to have explicit inherits in `Base_Test`. is it still the case?

Reply:
A better structure might be this since some contract may be required in Modifiers.

```mermaid
flowchart TD
Modifiers --> Fuzzers
Fuzzers --> Constants
Fuzzers --> Utils
Base --> Modifiers
```

Because Modifiers are the contac